src/components/ItemGrid.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="row">
+    <app-item v-for="(item, index) in items" @click.native="deleteItem(index)">{{ item }}</app-item>
+  </div>
+</template>
+
+<script>
+import Item from "./Item.vue";
+
+export default {
+  props: ["items"],
+  components: {
+    appItem: Item
+  },
+  methods: {
+    deleteItem(index) {
+      this.$emit("itemDeleted", index);
```

Comment:
Dobře, akorát se píše jinej casing
emituješ `item-deleted`. A i ten attribut v parent komponentě by měl bejt kebab-case

Jenom dodam že emitování funguje dobře pokud jdeš jen o jednu komponentu výš. Když už to jde přes víc komponent, je lepší to prohnat přes store.